airflow/jobs.py

```diff
@@ -723,7 +723,7 @@ def manage_slas(self, dag, session=None):
                         emails |= set(get_email_address_list(task.email))
                     elif isinstance(task.email, (list, tuple)):
                         emails |= set(task.email)
-            if emails and len(slas):
+            if emails:
```

Comment:
This line is in the if-check block of `if slas:` (https://github.com/apache/airflow/blob/fc3b45a61ac51441c3b9e4d99a20473cd3664056/airflow/jobs.py#L673). It's impossible for `len(slas)` to be zero here (correct me if I missed anything here)